### PR TITLE
Fix contributing link in README

### PR DESCRIPTION
**Pull request by ivoanjo**

### CODE REVIEW

**Comment by k0kubun** [comment]
Thanks :+1:

File: README.md [hunk 1]
```diff
@@ -379,7 +379,7 @@ Comparison:
 
 ## Contributing
 
-Bug reports and pull requests are welcome on GitHub at https://github.com/k0kubun/benchmark_driver.
+Bug reports and pull requests are welcome on GitHub at https://github.com/benchmark-driver/benchmark_driver.
```
Comment (eregon) [inline README.md]: It should be https://github.com/benchmark-driver/benchmark-driver actually now :smile:  (although GitHub redirects fine)

Comment (k0kubun) [reply README.md]: Haha, true :slightly_smiling_face:  https://github.com/benchmark-driver/benchmark-driver/commit/6ee9662953b29fd503ee2214ae4fbd08230a125d

Comment (ivoanjo) [reply README.md]: Ooops, totally missed it!

**Pull request merged**